samples/cpp/kalman.cpp

```diff
@@ -39,7 +43,9 @@ int main(int, char**)
 
     for(;;)
     {
-        randn( state, Scalar::all(0), Scalar::all(0.1) );
+        img = Scalar::all(0);
+        state.at<float>(0) = 0.0f;
+        state.at<float>(1) = 2.f * 3.14159f / 6;
```

Comment:
I recommend to use CV_PI or PI constant.

Reply:
fixed using `CV_PI`